### PR TITLE
kyverno added

### PR DESCRIPTION
**Pull request by evalsocket**
[
Kyverno](https://github.com/nirmata/kyverno) is a policy engine designed for Kubernetes. This PR add [kyverno-cli](https://github.com/nirmata/kyverno/blob/master/documentation/kyverno-cli.md) as a krew plugin to krew index.

### CODE REVIEW

**Comment by ahmetb** [comment]
Thanks for your contribution.
Why the `-cli` suffix? Isn't it clear when someone installs a kubectl plugin that it's a CLI tool?

File: plugins/kyverno.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  homepage: https://github.com/nirmata/kyverno
+  platforms:
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/nirmata/kyverno/releases/download/v1.1.6/kyverno-cli_v1.1.6_linux_x86_64.tar.gz
+    sha256: 61560c57ec5eb7f7c9da0d3b8331d90c02b593f8a2a86db86714c7e34f8eb91d
+    files:
+    - from: kyverno
```
Comment (corneliusweig) [inline plugins/kyverno.yaml:16]: Please also install the `LICENSE` file along with your executable.
(same in other places)

I see that the license is already part of your tarball. The easiest way to install everything that's in there is to remove the `files` section, as it defaults to `{from: "*", to: "."}`.

File: plugins/kyverno.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    sha256: 8fa665a256295bc090471e2b3b2545c19d34dcc74a744ff81a8a05f4a9bd4066
+    files:
+    - from: kyverno.exe
+      to: .
+    bin: kyverno.exe
+  shortDescription: Toolkit for testing kyverno policy
+  description: |+2
+    The Kyverno Command Line Interface (CLI) is designed to validate policies and test the behavior of applying policies to resources before adding the policy to a cluster. It can be used as a kubectl plugin and as a standalone CLI.
+  caveats: |
+    The plugin requires access to create Policy and CustomResources
```
Comment (corneliusweig) [inline plugins/kyverno.yaml:43]: Can you be more specific about the kind of `Policy`?

File: plugins/kyverno.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        os: windows
+        arch: amd64
+    uri: https://github.com/nirmata/kyverno/releases/download/v1.1.6/kyverno-cli_v1.1.6_windows_x86_64.zip
+    sha256: 8fa665a256295bc090471e2b3b2545c19d34dcc74a744ff81a8a05f4a9bd4066
+    files:
+    - from: kyverno.exe
+      to: .
+    bin: kyverno.exe
+  shortDescription: Toolkit for testing kyverno policy
+  description: |+2
```
Comment (corneliusweig) [inline plugins/kyverno.yaml:40]: Please wrap at ~80 chars, so that it prints nicely on wide and narrow terminals.

File: plugins/kyverno.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    The Kyverno Command Line Interface (CLI) is designed to validate policies and test the behavior of applying policies to resources before adding the policy to a cluster. It can be used as a kubectl plugin and as a standalone CLI.
```
Comment (corneliusweig) [inline plugins/kyverno.yaml:41]:
```suggestion
    The Kyverno Command Line Interface (CLI) is designed to validate policies and test the behavior of applying policies to resources before adding the policy to a cluster.
```
The last bit is redundant for sure.

Comment (corneliusweig) [inline plugins/kyverno.yaml:41]: My personal taste would be to give more context about `kyverno`. Maybe give an example or describe a problem that can only be solved with kyverno.

File: plugins/kyverno.yaml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    uri: https://github.com/nirmata/kyverno/releases/download/v1.1.6/kyverno-cli_v1.1.6_windows_x86_64.zip
+    sha256: 8fa665a256295bc090471e2b3b2545c19d34dcc74a744ff81a8a05f4a9bd4066
+    files:
+    - from: kyverno.exe
+      to: .
+    - from: LICENSE
+      to: .
+    bin: kyverno.exe
+  shortDescription: Kyverno is a policy engine for kubernetes 
+  description: |+2
```
Comment (corneliusweig) [inline plugins/kyverno.yaml:46]: Please wrap at ~80 chars.

Comment (corneliusweig) [inline plugins/kyverno.yaml:46]: Before kyverno can be used, you need to install the operator, right?  (from https://github.com/nirmata/kyverno#quick-start)
Folks usually mention such steps in their `caveats`, which makes a lot of sense.

In addition, the original draft mentioned what rights are required to run `kyverno` (Policy & CustomResource). It makes sense to mention this in `caveats`, too. I presume `CustomResource` is only required for setup? If so, you can mention that as well.

File: plugins/kyverno.yaml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+     Kyverno is a policy engine that provides simplified validation, mutation, and generation of resource configurations. Kyverno cli can be used to test policies and apply policies to resources files
```
Comment (corneliusweig) [inline plugins/kyverno.yaml:47]:
```suggestion
     Kyverno is a policy engine that provides simplified validation, mutation, and generation of resource configurations. Kyverno can be used to test policies and apply policies to resources files
```

**Comment by corneliusweig** [comment]
/lgtm
/approve
Thanks for contributing!

**Pull request merged**